src/components/Home/WalletCard.tsx

```diff
@@ -75,13 +74,20 @@ export function WalletCard({ type, onClick, ...props }: WalletCardProps) {
                             peanut.me/<span className="font-bold">{username}</span>
                         </p>
                     </div>
-                    <p className="text-4xl font-black sm:text-5xl">
-                        $ {Number(formatUnits(wallet.balance, 6)).toFixed(2)}
-                    </p>
-                    <div>
+                    <p className="text-4xl font-black sm:text-5xl">$ {printableUsdc(wallet.balance)}</p>
+                    <div className="relative">
                         <div className="flex flex-col">
                             <p className="text-xl font-black sm:text-2xl">{shortenAddressLong(wallet.address)}</p>
                         </div>
+                        <Icon
+                            name="content-copy"
+                            className="absolute bottom-0 right-0 h-5 w-5 hover:opacity-80"
+                            fill="white"
+                            onClick={(e: MouseEvent<SVGElement>) => {
+                                e.stopPropagation()
+                                navigator.clipboard.writeText(wallet.address)
+                            }}
+                        />
```

Comment:
In my opinion this is bad from a UX perspective - you click the copy thing, but nothing happens, visually. It's not clear that anything was copied.

We should make it clear for the user that smth happened. I'll make a discord thread

Reply:
added toast info